Month3/Project1/Project1/TStaticArray.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+template <typename T, int N>
+class TStaticArray
+{
+private:
+	int SizeArray = N;
+	T* Array = new T[N];
+public:
+	T& operator[](const int Index)
+	{
+	    return Array[Index];
+	}
+
+	const T& operator[](const int Index) const
+	{
+		return Array[Index];
+	}
+
+	T& Front() 
+	{
+		return Array[0];
+	}
+	const T& Front() const 
+	{
+		return Array[0];
+	}
+
+	T& Back()
+	{
+		return Array[SizeArray - 1];
+	}
+	const T& Back() const 
+	{
+		return Array[SizeArray - 1];
+	}
+
+	T& GetData() 
+	{
+		return Array;
+	}
+	const T& GetData() const 
+	{
+		return Array;
+	}
+
+	bool IsEmpty()
+	{
+		return SizeArray == 0 ;
+	}
+
+	void Fill(T& Filler)
```

Comment:
Remember: when do you want to use references and when do you want to use const references?